content/docs/components/data-display/card/index.mdx

```diff
@@ -0,0 +1,296 @@
+# Card
+
+## Import
+
+Chakra UI exports 4 components to help you create a Card.
+
+```js
+import { Card, CardHeader, CardBody, CardFooter } from '@chakra-ui/react';
+```
+
+- **Card:** The parent wrapper that provides context for its children.
+- **CardHeader:** The wrapper that contains a card's header.
+- **CardBody:** The wrapper that houses the card's main content.
+- **CardFooter:** The footer that houses the card actions.
+
+## Usage
+
+### Basic Card
+
+Put in some content in the `CardBody` to get a basic card.
+
+```jsx live
```

Comment:
You might want to check if the examples are rendered properly. I've added a way to make the live preview box different sizes so you don't have scrolling examples

Reply:
I'm not sure I get what you mean. The examples render correctly with the `live` tag, although there's a scroll in the preview. In what way did you make the examples non-scrolling?

Reply:
If you check other components live blocks in the code. There is in some places sm,md,lg to describe which high it should have. This is because some components are to big for the default high of the iframe.